src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -199,14 +199,15 @@ def remove_sync_policy_group(self, group_id: str, bucket_name=''):
     @Endpoint(method='PUT', path='/sync-flow')
     @EndpointDoc("Create or update the sync flow")
     @CreatePermission
+    # pylint: disable=W0102
```

Comment:
Just a NIT, but I think instead of having to disable pylint (which is there for a reason) we should try to follow the best practices

https://pylint.pycqa.org/en/latest/user_guide/messages/warning/dangerous-default-value.html